docs/pages/guides/typescript.mdx

```diff
@@ -145,6 +131,42 @@ Consider the following when using path aliases:
 - Path aliases are only supported by Metro (including Metro web) and not by `@expo/webpack-config`.
 - Bare projects require additional setup for this feature. See the [versioned Metro setup guide](/versions/latest/config/metro#bare-workflow-setup) for more information.
 
+<Tabs>
+
+<Tab label="SDK 50 and above">
+
+`tsconfigPaths` is enabled by default. You can disable it by setting `tsconfigPaths` to `false` in the project's [app config](/workflow/configuration/):
+
+```json app.json
+{
+  "expo": {
+    "experiments": {
+      "tsconfigPaths": false
+    }
+  }
+}
+```
+
+</Tab>
+
+<Tab label="SDK 49">
+
+In SDK 49 projects, you'll need to enable path aliases in the project's [app config](/workflow/configuration/):
```

Comment:
```suggestion
Set `tsconfigPaths` to `true` to enable path aliases in the project's [app config](/workflow/configuration/):
```

Let's move the redundancy of repeating SDK 49 here since the tab already says these instructions are for SDK 49.